templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.Mongo/server/mongo/settings.py

```diff
@@ -0,0 +1,14 @@
+import os
+from os.path import join, dirname, realpath
+from dotenv import load_dotenv
+
+#create .env file path
+dotenv_path = join(dirname(dirname(dirname(realpath(__file__)))), '.env')
+
+#load file from path
+load_dotenv(dotenv_path)
```

Comment:
Out of the scope of this PR, but it seems to me that using `.env` files for secrets is not the best security practice. Of course, it is difficult to not use this but also make the resulting app work in different environments. If the app is to be deployed in Azure we could use a Key Vault. At the very least we should add `.env` to a generated `.gitignore`

Reply:
I agree. To add .env to a .gitignore file, would it make sense to generate a .gitignore if we are not making a git repository for the generated project? Just wondering if it confusing to the user or is a normal behavior?

Reply:
Yeah, we could TODO / readme item to explicitly say "hey, user, take care of secrets management" ;)